### ISSUE TITLE
Has this project been abandoned?(!)

### ISSUE THREAD

**Issue by bhargavaman** [sidebase/create-sidebase#35]
### Ask your question

Haven't seen any changes / blog posts about it  in a while, just curious! Love the work you guys are putting in on nuxt-auth, but would love if this project was up to date with the latest modules as well. 

Follow up - are you guys still open to PRs?

**Comment by zoey-kaiser** [comment]
Hi @bhargavaman 👋 

We have been a little bit less active within the last half year, as our resources have been pretty stretched. However, we committed to allocating more resources to work again on sidebase this year!

We have decided to prioritize the following projects:

- Picking up development on v1.0 of NuxtAuth (read more [here](https://github.com/sidebase/nuxt-auth/issues/673))
- Begun working on a rewrite and v1.0 of NuxtPDF (read more [here](https://github.com/sidebase/nuxt-pdf/issues/25))

We have also begun discussing internally how we want to proceed with `create-sidebase`. I am currently still working on an outline for the planned changes, but I assume it will contain:
- Deprecating the cheviot stack
- Ramping up mantaince on the merino stack to ensure it always stays up to date
- Adding a new unnamed stack, that will allow you to configure even more!
  - `merino` aims to be as simple as possible, however, we want to add another stack with more modules and different options for eg. Database providers (Prisma, Drizzle etc.) as well as many other nice setup perks to have!
  - We will avoid just adding Nuxt modules, that can be installed with the Nuxt CLI, as we see this as a bit unnecessary, therefore we will focus on integrating modules and packages that require more setup than just the one command!
  - However, this will lead to this stack being more opinionated, as we enforce our best practices! 

> Follow up - are you guys still open to PRs?

Yes, we are! Like I mentioned above, we are currently going through our projects (based of popularity) and working on them one by one, so it may take longer to merge a PR on eg. `create-sidebase` which is lower down on our priority list. However, feel free to ping me in the PR, so I receive a notification and can look into it 😄
